packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -341,6 +342,11 @@ class FlutterCommandRunner extends CommandRunner<void> {
             status = version.toString();
           }
           globals.printStatus(status);
+          globals.analytics.send(Event.flutterCommandResult(
```

Comment:
nit. can you put this line right after line 332, and also pass `FlutterGlobalOptions.kVersionFlag` as the commandPath to make it more obvious this is replicating line 332?